frontend/fingertips-frontend/app/page.tsx

```diff
@@ -10,51 +10,22 @@ type Forecast = {
 
 export default async function Home() {
   // We don't want to render this page statically
-  await connection();
+  // await connection();
 
-  const apiUrl = process.env.FINGERTIPS_API_URL;
+  // const apiUrl = process.env.FINGERTIPS_API_URL;
 
-  if (!apiUrl) {
-    throw new Error(
-      "No API URL set. Have you set the FINGERTIPS_API_URL environment variable?"
-    );
-  }
+  // if (!apiUrl) {
+  //   throw new Error(
+  //     'No API URL set. Have you set the FINGERTIPS_API_URL environment variable?'
+  //   );
+  // }
 
-  const weatherData = await fetch(apiUrl, {
-    // Cache the data for 60s
-    next: { revalidate: 60 },
-  });
-  const forecasts: Forecast[] = await weatherData.json();
+  // const weatherData = await fetch(apiUrl, {
+  //   // Cache the data for 60s
+  //   next: { revalidate: 60 },
+  // });
+  // const forecasts: Forecast[] = await weatherData.json();
 
-  return (
-    <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
-      <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
-        <div className="p-8">
-          <Image
-            className="float-right"
-            src="/fingertips-logo.svg"
-            alt="Fingertips logo"
-            width={240}
-            height={140}
-            priority
-          />
-
-          <h1>FingertipsNext</h1>
-
-          <p>
-            Fingertips is a large public health data collection. Data is
-            organised into themed profiles.
-          </p>
-
-          <ul>
-            {forecasts.map((f) => (
-              <li key={f.date}>
-                {f.date}, {f.temperatureC}, {f.temperatureF}, {f.summary}
-              </li>
-            ))}
-          </ul>
-        </div>
-      </main>
-    </div>
-  );
+  // return <HomePage forecasts={forecasts} />;
```

Comment:
We have commented this out for now in order to get the cypress tests running in the pipeline.

Reply:
fixed for now by getting the cypress test to go straight to `/search` and bypassing the homepage. But we do want to think about how we create a test that calls the API and how we would go about mocking this call?